schemas/session-replay/mobile/image-wireframe-update-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/mobile/image-wireframe-update-schema.json",
+  "title": "ImageWireframeUpdate",
+  "type": "object",
+  "description": "Schema of all properties of a ImageWireframeUpdate.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-update-schema.json"
+    },
+    {
+      "required": ["type"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "image",
+          "readOnly": true
+        },
+        "base64": {
+          "type": ["string", "null"],
+          "description": "base64 representation of the image. Can have a null value",
+          "readOnly": false
+        },
+        "imageFileExtension": {
+          "type": "string",
+          "description": "Extension of the image file",
+          "readOnly": false
+        }
+      }
```

Comment:
**question:** we can change the image content and/or format? For my it makes little sense

Reply:
The idea is to start with empty image wireframe (just size) and provide base64 after we calculate it on the SDK side.

Reply:
Yes but what @ThibautGeriz says makes sense, you will only change the content and not the format (file extension)